bin/console-conf-wrapper

```diff
@@ -79,4 +79,8 @@ fi
 
 cat /usr/share/subiquity/console-conf-wait
 read REPLY
+# start the console-conf routine in snapd to delay any new refreshes, and wait
+# for current refreshes to complete, this will print off messages if there are
+# on-going refreshes
+snap routine console-conf-start
```

Comment:
we are under `set -e` at this point. will this ever fail with non-zero code?

currently this routine not available in stable or beta.